.github/workflows/runners.yaml

```diff
@@ -25,97 +27,36 @@ on:
       - '!**.md'
 
 env:
-  RUNNER_VERSION: 2.299.1
+  # Safeguard to prevent pushing images to registeries after build  
+  PUSH_TO_REGISTRIES: true
+  TARGET_ORG: actions-runner-controller
+  TARGET_WORKFLOW: release-runners.yaml
```

Comment:
hi @Link- 👋

I tried to find this yaml but failed. Is this file supposed to be on `master` already?

Reply:
We had a major overhaul of the release workflows and we had to split the runtime over 2 repos. 

That file is here: https://github.com/actions-runner-controller/releases/tree/main/.github/workflows

Reply:
I wasn't aware of this maintained fork, so this change might be breaking to you I'm afraid. You might want to revert the changes to the release file and use the old ones instead.

Reply:
Hi, thanks for the replies and no worries! This is more of a PoC, so we can do GPU stuff using Github Actions. I'll have a look and see what makes sense :+1:

Reply:
added it here instead: https://github.com/ddelange/actions-runner-controller-releases